src/main/java/org/eclipse/yasson/internal/model/JsonbAnnotated.java

```diff
@@ -1,78 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2016, 2017 Oracle and/or its affiliates. All rights reserved.
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License v1.0 and Eclipse Distribution License v. 1.0
- * which accompanies this distribution.
- * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
- * and the Eclipse Distribution License is available at
- * http://www.eclipse.org/org/documents/edl-v10.php.
- *
- * Contributors:
- * Roman Grigoriadi
- ******************************************************************************/
-
-package org.eclipse.yasson.internal.model;
-
-import java.lang.annotation.Annotation;
-import java.lang.reflect.AnnotatedElement;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.Map;
-
-import javax.json.bind.JsonbException;
-
-import org.eclipse.yasson.internal.properties.MessageKeys;
-import org.eclipse.yasson.internal.properties.Messages;
-
-/**
- * Element wrapper containing merged annotation from class, superclasses and interfaces.
```

Comment:
Was this object never used to store JSON-B annotation data about classes/interfaces?

Reply:
I checked, it's being used for classes in AnnotationIntrospector, so it makes sense.